parsers/ENTSOE.py

```diff
@@ -1001,14 +941,6 @@ def validate_production(
 
     if validation_criteria:
         return validate(datapoint, logger=logger, **validation_criteria)
-
-    # NOTE: Why are there sepcial checks for these zones?
-    if zone_key.startswith("DK-"):
```

Comment:
Aren't we removing all validation for these zones?

Reply:
They don't have any other validation going on so it shouldn't be a problem.